### PR TITLE
Size reduced

### LINKED ISSUE #1915: APK size too big.
**Actual Behaviour**

APK size is 30.2 MB

**Expected Behaviour**

It should be lesser.

**Steps to reproduce it**

1. Add option "minifyEnabled true" to make the proguard enabled.
2. Convert PNG images to WebP images.
3. Added a proguard File.

**Would you like to work on the issue?**

Yes, I would like to work on the issue.

### PR DESCRIPTION
**Pull request by DhruvamSharma**
Fixed #1915  

Changes: Added a proguard file. 
               Converted PNG images to WebP images.
               Added the option to minifyEnabled.

### CODE REVIEW

**Comment by sridharjajoo** [comment]
@DhruvamSharma how did you check the size of the APK after applying proguard.You must have a signed APK for testing that.

**Comment by DhruvamSharma** [comment]
Yes @sridharjajoo I signed the APK for testing that.

File: app/src/main/res/menu/main.xml [hunk 1]
```diff
@@ -1,10 +1,10 @@
-<menu xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:android="http://schemas.android.com/apk/res/android" >
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
```
Comment (abishekvashok) [inline app/src/main/res/menu/main.xml]: Why do we go for this such change.

**Comment by DhruvamSharma** [comment]
@abishekvashok @sridharjajoo @yashLadha Please see the requested changes in PR #1950

**Comment by yashLadha** [comment]
Please try to follow the best practices and do not add commits of other PR's into this.

**Comment by DhruvamSharma** [comment]
I am really sorry. I am still learning advanced git usage. So I'll make
sure I don't do that again.

**Comment by yashLadha** [comment]
Try to work on different branched and reset the head to the main branch and always ammend the commit, this way you will escape from rebasing.

**Comment by DhruvamSharma** [comment]
Thank youuu

**Pull request closed without merging**